docker-compose.yml

```diff
@@ -1087,7 +1087,6 @@ services:
   appapi-dsp:
     image: ghcr.io/nextcloud/nextcloud-appapi-dsp:release
     container_name: nextcloud-appapi-dsp-http
```

Comment:
The container name can probably also be dropped

Reply:
can be useful for easy `docker inspect` sometimes but yeah not really required